### ISSUE TITLE
Chaos Sanctuary

### ISSUE THREAD

**Issue by redd990** [kolton/d2bot-with-kolbot#3204]
Hey. 
I wouldnt say its an issue, but sometimes all the charsdont enter the chaos at the same time. Im having issues with the characters that dont teleport. They seem to run directly for the seal first and then turn around to start clearing the  monsters.  In those say 5-20 seconds of trying to push thru dudes, they get hit. 

Is there anyway to make the non-tele/enigma characters to recognize and attack quicker??
Thanks!

**Comment by bmarc94** [comment]
Is diablo config mode fast is set to false?

**Comment by redd990** [comment]
Yes it was and is. I got'er dialed now, my main tele char was an Hdin,  who had dodge  = true; Would explain why my sorcs were always chickenning and then dying.
Now Pally dodge = false; 
sorc 1
both dodge = true ; alwaysdodge(100) distance i change often between 8 and 15  
sorc 2
Sorry late reply, didnt think to look for notifications 📦 
P"EACE
